src/bindings/python/requirements_test.txt

```
@@ -25,6 +25,7 @@ mypy
 onnx<1.12.0
 Pep8-naming
 pydocstyle
+pytest-forked
```

```suggestion
pytest-forked; platform_system != "Windows"
```
How about this restriction?